pom.xml

```diff
@@ -23,7 +23,7 @@
 
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.289.1</jenkins.version>
+        <jenkins.version>2.332.3</jenkins.version>
```

Comment:
Was there a specific reason to use 2.332.3 rather than 2.332.1? https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version

Reply:
Not really, I somehow had in mind that the latest point release of an LTS version is the desired version.
If needed, we can use .1 here, the relevant change has been introduced in 2.321 iirc.

Reply:
We went back and forth but this was the compromise position from https://github.com/jenkins-infra/jenkins.io/pull/4876.

Reply:
Oic, thanks for the clarification.